app/src/components/features/rules/GettingStarted/WorkspaceOnboarding/OnboardingSteps/TeamWorkspaces/WorkspaceCreation.tsx

```diff
@@ -90,6 +93,9 @@ export const CreateWorkspace: React.FC<Props> = ({ defaultTeamData }) => {
               is_admin: makeUserAdmin,
               source: "onboarding",
               num_users_added: inviteEmails.length,
+              workspace_type: isWorkspaceMappedToBillingTeam(defaultTeamData?.teamId ?? newTeamId, billingTeams)
+                ? "mapped_to_billing_team"
+                : "not_mapped_to_billing_team",
```

Comment:
since these are being used this often, let's convert these into constants and then use them

Reply:
++